runtime/onert/core/src/util/shapeinf/Fill.cc

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+#include "ir/Data.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+ir::Shape inferFillShape(const ir::Shape &in_shape, const int32_t *buffer)
+{
+  ir::Shape out_shape(in_shape.dim(0));
+
+  for (int out_x = 0; out_x < out_shape.rank(); ++out_x)
+  {
+    out_shape.dim(out_x) = buffer[out_x];
+  }
+
+  return out_shape;
+}
+
+void StaticInferer::visit(const ir::operation::Fill &op)
+{
+  const auto input_idx{op.getInputs().at(ir::operation::Fill::Input::INPUT)};
+  const auto &input = _operands.at(input_idx);
+  const auto output_idx = op.getOutputs().at(0);
+  ir::Operand &output = _operands.at(output_idx);
+
+  if (input.info().isDynamic())
+  {
+    output.info().setDynamic();
+    return;
+  }
+
+  if (!input.isConstant())
+  {
+    output.info().setDynamic();
+    return;
+  }
+
```

Comment:
Could you add checking data type of `input`?

Reply:
Please refer to https://www.tensorflow.org/api_docs/python/tf/fill
Fill op doesn't support uint32 type.